components/skus/browser/resources/skus_internals.tsx

```diff
@@ -3,12 +3,58 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
+import * as SkusInternalsMojo from 'gen/brave/components/skus/common/skus_internals.mojom.m.js'
 import * as React from 'react'
+import styled from 'styled-components'
 import { render } from 'react-dom'
+import { JsonView } from 'react-json-view-lite';
+import 'react-json-view-lite/dist/index.css';
+
+const Container = styled.div`
+  margin: 4px;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+`
+
+const ButtonContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: 4px;
+`
+
+const API = SkusInternalsMojo.SkusInternals.getRemote()
 
 function App() {
+  const [skusState, setSkusState] = React.useState({})
+
+  const resetSkusState = () => {
+    API.resetSkusState()
+    setSkusState({})
+  }
+
+  const getSkusState = () => {
+    API.getSkusState().then((r: any) => {
+      setSkusState(JSON.parse(r.response))
+    })
+  }
+
   return (
-    <div>Log here</div>
+    <Container>
+      <h2>Skus internals</h2>
+      <ButtonContainer>
+        <button onClick={() => {
+            if (window.confirm('Resetting SKUs state will count the next sign in for account.brave.com as a new device. If you do this more than once you may run out of devices/credentials. Are you sure you want to reset SKUs state?')) {
```

Comment:
are inline scripts allowed to run on this page or is this pre-processed into an external javascript file? seems worth checking that the page has a content security policy which disallows unsafe-inline script.

Reply:
you should be able to check this by looking at the content-security-policy header in devtools upon loading the page

Reply:
`CSP` doesn't have `unsafe-inline`.
```
child-src 'none';object-src 'none';require-trusted-types-for 'script';script-src chrome://resources 'self';trusted-types default;frame-ancestors 'none';
```
<img width="1423" alt="image" src="https://github.com/brave/brave-core/assets/6786187/39e91be5-98db-4acf-b31f-6bd0d728f609">

Reply:
@diracdeltas any other concerns?

Reply:
lgtm. thanks.